package.json

```diff
@@ -81,11 +81,12 @@
     }
   },
   "dependencies": {
-    "@nestjs/axios": "0.1.0",
-    "@nestjs/common": "9.3.11",
-    "@nestjs/core": "9.3.11",
-    "@nestjs/platform-express": "8.4.7",
+    "@nestjs/axios": "3.0.1",
+    "@nestjs/common": "10.2.9",
+    "@nestjs/core": "10.2.9",
+    "@nestjs/platform-express": "10.2.9",
     "@nuxtjs/opencollective": "0.3.2",
+    "axios": "1.6.0",
```

Comment:
why do you set a specific `axios` version here. `@nestjs/axios` is using `1.6.2` issn't this just make this harder to maintain?

Reply:
If I remove axios, an warning occurs like below.
> warning " > @nestjs/axios@3.0.1" has unmet peer dependency "axios@^1.3.1"

Is it okay to ignore this, or should I add axios to devDependencies?

Reply:
as this is a library I would say this warning is okay as the consuming package should install its axios version

Reply:
I see, I removed `axios` from dependencies